### PR TITLE
4.13 release notes for Node and MCO

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by rioliu-rh** [comment]
@mburke5678 for mco part lgtm

**Comment by mburke5678** [comment]
@sunilcio CAn you PTAL at the [release notes for the Nodes features](https://github.com/openshift/openshift-docs/pull/57061/files#diff-5a1bef860ba4f103b12cdc2db661140fa691167a8b50abea180008754482951cR178-R207)? Thank you in advance!!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -160,9 +160,51 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
...
 === Nodes
 
+[id="ocp-4-13-nodes-mirror"]
+==== Image registry repository mirroring by tags
+
+You can now pull images from a mirrored registry by using image tags and not only digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` object (ITMS) to pull images by using image tags.
+
+If you have existing YAML files that you used to create ICSP objects, you can use the `oc adm migrate icsp` command to convert those files to an IDMS YAML file.
+
+For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
```
Comment (lyman9966) [inline release_notes/ocp-4-13-release-notes.adoc:187]: It seems the link for "For more information on these new objects, see" is not latest. It should be : https://56250--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -160,9 +160,51 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
...
+
+For more information on converting existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
```
Comment (lyman9966) [inline release_notes/ocp-4-13-release-notes.adoc:189]: The link of "For more information on converting existing ICSP objects to IDMS objects, see" not exist. It should be: https://56250--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror-convert_post-install-preparing-for-users

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (mburke5678) [reply release_notes/ocp-4-13-release-notes.adoc:255]: I rebased the branch to bring down the new files referenced in the links. The links as they are should work properly.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -228,9 +228,44 @@ For more information, see xref:../machine_management/control_plane_machine_manag
...
+You can now pull images from a mirrored registry by using image tags in addition to digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` (ITMS) object to pull images by using image tags.
+
+If you have existing YAML files that you used to create ICSP objects, you can use the `oc adm migrate icsp` command to convert those files to an IDMS YAML file.
+
+For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
+
+For more information on converting existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
+
+[id="ocp-4-13-nodes-crun-ga"]
+==== crun general availability
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:260]: Does crun have to be in `` or pascal case?

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -228,9 +228,44 @@ For more information, see xref:../machine_management/control_plane_machine_manag
...
+
+For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#core-user-password_post-install-machine-configuration-tasks[Changing the core user password for node access].
+
 [id="ocp-4-13-nodes"]
 === Nodes
 
+[id="ocp-4-13-nodes-mirror"]
+==== Image registry repository mirroring by tags
+
+You can now pull images from a mirrored registry by using image tags in addition to digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` (ITMS) object to pull images by using image tags.
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:251]:
```suggestion
You can now pull images from a mirrored registry by using the image tags in addition to digest specifications. To accomplish this change, the `ImageContentSourcePolicy` (ICSP) object is deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using the digest specifications or an `ImageTagMirrorSet` (ITMS) object to pull images by using image tags.
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -228,9 +228,44 @@ For more information, see xref:../machine_management/control_plane_machine_manag
...
+
+If you have existing YAML files that you used to create ICSP objects, you can use the `oc adm migrate icsp` command to convert those files to an IDMS YAML file.
+
+For more information on these new objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror_post-install-preparing-for-users[Configuring image registry repository mirroring].
+
+For more information on converting existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:257]:
```suggestion
For more information on converting the existing ICSP objects to IDMS objects, see xref:../post_installation_configuration/preparing-for-users.adoc#images-configuration-registry-mirror-convert_post-install-preparing-for-users[Converting ImageContentSourcePolicy (ICSP) files for image registry repository mirroring].
```

File: release_notes/ocp-4-13-release-notes.adoc [hunk 6]
```diff
@@ -510,6 +560,11 @@ CoreDNS will stop supporting wildcard DNS queries for names under the `cluster.l
 
 In {product-title} 4.12, support for Kuryr on clusters that run on {rh-openstack} is deprecated. Support will be removed no earlier than {product-title} 4.14.
 
+[id="ocp-4-13-icsp"]
+==== `ImageContentSourcePolicy` objects
+
+The `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using digest specifications or an `ImageTagMirrorSet` object (ITMS) to pull images by using image tags.
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:566]:
```suggestion
The `ImageContentSourcePolicy` (ICSP) object is now deprecated. You can now use an `ImageDigestMirrorSet` (IDMS) object to pull images by using the digest specifications or an `ImageTagMirrorSet` object (ITMS) to pull images by using image tags.
```

**Review by snarayan-redhat — COMMENTED** [review]
Looks great! Just few nits.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 7]
```diff
@@ -202,7 +202,7 @@ For more information about assigning IP addresses from an IP address pool to ser
 === Storage
 
 [id="ocp-4-13-olm"]
-=== Operator lifecycle
+=== Operator lifecycleImageContentSourcePolicy` (ICSP) object
```
Comment (snarayan-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:205]:
```suggestion
=== Operator lifecycle `ImageContentSourcePolicy` (ICSP) object 
```

File: release_notes/ocp-4-13-release-notes.adoc (hunk 3 above)
Comment (mburke5678) [reply release_notes/ocp-4-13-release-notes.adoc:260]: @snarayan-redhat As I understand, crun is a proper name that doesn't need mark-up and is always lower case. https://github.com/containers/crun

**Comment by mburke5678** [comment]
@yuqi-zhang

**Pull request merged**